ext/mini_racer_extension/mini_racer_extension.cc

```diff
@@ -949,6 +958,10 @@ static VALUE rb_isolate_pump_message_loop(VALUE self) {
     }
 }
 
+static void NoRubyErrorConstructor(const FunctionCallbackInfo<Value>& args) {
+    args.GetIsolate()->ThrowError("This class may not be constructed from JS!");
+}
```

Comment:
ThrowError is not a function in our current version of v8. ThrowException exists, but requires a Local<> Error instance.

Reply:
I've inlined the definition as it appears in my version of V8.